FoundationToSoil.cs

```diff
@@ -57,7 +57,8 @@ static void Postfix(Player __instance)
 
             if ( fNumToConvert > 0)
             {
-                inventory.TakeItem(1131, fNumToConvert);
+                int dummy;
+                inventory.TakeItem(1131, fNumToConvert, out dummy);
```

Comment:
You can combine the declaration and assignment here:
```suggestion
                inventory.TakeItem(1131, fNumToConvert, out int dummy);
```